docs/commands.md

```diff
@@ -13,152 +13,178 @@ Executes tests. Requires `codecept.conf.js` config to be present in provided pat
 Run all tests from current dir
 
 ```sh
-codeceptjs run
+npx codeceptjs run
 ```
 
 Load config and run tests from `test` dir
 
 ```sh
-codeceptjs run -c test
+npx codeceptjs run -c test
 ```
 
 Run only tests with "signin" word in name
 
 ```sh
-codeceptjs run --grep "signin"
+npx codeceptjs run --grep "signin"
 ```
 
 Run all tests without "@IEOnly" word in name
 
 ```sh
-codeceptjs run --grep "@IEOnly" --invert
+npx codeceptjs run --grep "@IEOnly" --invert
 ```
 
 Run single test [path to codecept.js] [test filename]
 
 ```sh
-codeceptjs run github_test.js
+npx codeceptjs run github_test.js
 ```
 
 Run single test with steps printed
 
 ```sh
-codeceptjs run github_test.js --steps
+npx codeceptjs run github_test.js --steps
 ```
 
 Run single test in debug mode
 
 ```sh
-codeceptjs run github_test.js --debug
+npx codeceptjs run github_test.js --debug
 ```
 
 Run test with internal logs printed (global promises, and events).
 
 ```sh
-codeceptjs run github_test.js --verbose
+npx codeceptjs run github_test.js --verbose
 ```
 
 Select config file manually (`-c` or `--config` option)
 
 ```sh
-codeceptjs run -c my.codecept.conf.js
-codeceptjs run --config path/to/codecept.json
+npx codeceptjs run -c my.codecept.conf.js
+npx codeceptjs run --config path/to/codecept.json
 ```
 
 Override config on the fly. Provide valid JSON which will be merged into current config:
 
 ```sh
-codeceptjs run --override '{ "helpers": {"WebDriver": {"browser": "chrome"}}}'
+npx codeceptjs run --override '{ "helpers": {"WebDriver": {"browser": "chrome"}}}'
 ```
 
 Run tests and produce xunit report:
 
 ```sh
-codeceptjs run --reporter xunit
+npx codeceptjs run --reporter xunit
 ```
 
 Use any of [Mocha reporters](https://github.com/mochajs/mocha/tree/master/lib/reporters) used.
 
+## Dry Run
+
+Prints test scenarios without executing them
+
+```
+npx codeceptjs dry-run
+```
+
+When passed `--steps` or `--debug` option runs tests, disabling all plugins and helpers, so you can get step-by-step report with no tests actually executed.
+
+```
+npx codeceptjs dry-run --steps
+```
+
+If a plugin needs to be enabled in `dry-run` mode, pass its name in `-p` option:
+
+```
+npx codeceptjs dry-run --steps -p allure
+```
+
+To enable bootstrap script in dry-run mode, pass in `--bootstrap` option when running with `--steps` or `--debug`
+
+```
+nox codeceptjs dry-run --steps --bootstrap
```

Comment:
npx instead of nox

Reply:
yes, thank you. Could you make a pull request fixing this?

Reply:
Sure